.github/workflows/pr.yml

```diff
@@ -562,9 +563,22 @@ jobs:
             VSC_CI_MATRIX_TEST_SUITE: ${{matrix.test-suite}}
             CI_JUPYTER_FAST_COMPILATION: 'true'
 
-
       # Used by tests for non-python kernels.
       # Test are enabled via env variable `VSC_JUPYTER_CI_RUN_NON_PYTHON_NB_TEST`
+      - name: Install Dot.net
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: ${{env.DOTNET_VERSION}}
+        if: matrix.test-suite != 'notebookWithoutPythonExt'
+
+      - name: Install .NET Interactive
+        run: dotnet tool install -g --add-source "https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" Microsoft.dotnet-interactive
+        if: matrix.test-suite != 'notebookWithoutPythonExt'
+
+      - name: Install .NET Kernel
+        run: dotnet interactive jupyter install
+        if: matrix.test-suite != 'notebookWithoutPythonExt'
```

Comment:
Should this be `if: matrix.test-suite == notebookWithoutPythonExt`? That test suite is for testing our notebooks without the Python extension, right?

Reply:
Nope this is correct. .NET kernel cannot be installed if you do not have Jupyter.